src/config/dynamic-theme-fixes.config

```diff
@@ -11915,6 +11915,18 @@ img[src*="libertesklep.svg"]
 
 ================================
 
+libgen.is
+libgen.rs
+libgen.st
+
+CSS
+.dropdown_3lettercolumn, .dropdown_1column, .dropdown_2columns, .dropdown_3columns, .dropdown_4columns, .dropdown_5columns, .dropdown_6columns {
+    background-image: -webkit-linear-gradient(rgb(37, 41, 43), rgb(24, 26, 27)) !important;
+    background-image: linear-gradient(rgb(37, 41, 43), rgb(24, 26, 27)) !important;
```

Comment:
Please [see the contribution guide](https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md#how-to-use-the-dev-tools) to check how to format colors.